tests/configs/without_modules.js

```diff
@@ -3,10 +3,11 @@
  * By Rodrigo Ramírez Norambuena https://rodrigoramirez.com
  * MIT Licensed.
  */
-let config = require(process.cwd() + "/tests/configs/default.js").configFactory({
-	ipWhitelist: ["127.0.0.1", "::ffff:127.0.0.1", "::1", "::ffff:192.168.10.1"]
-});
-delete config.modules;
+let config = {
+	ipWhitelist: ["127.0.0.1", "::ffff:127.0.0.1", "::1", "::ffff:192.168.10.1"],
+
+	units: "metric"
```

Comment:
do we need that unit config entry?

Reply:
no, did an automatic search/replace, will check again and update.